tests/contrib/gp/test_models.py

```diff
@@ -180,12 +178,9 @@ def test_inference(model_class, X, y, kernel, likelihood):
 
     gp.optimize(optim.Adam({"lr": 0.01}), num_steps=1000)
 
-    variance = gp.kernel.get_param("variance")
-    lengthscale = gp.kernel.get_param("lengthscale")
-    target_variance = kernel.get_param("variance")
-    target_lengthscale = kernel.get_param("lengthscale")
-    assert_equal(variance, target_variance, prec=0.2)
-    assert_equal(lengthscale, target_lengthscale, prec=0.2)
+    y_cov = gp.kernel(X)
+    target_y_cov = kernel(X)
+    assert_equal(y_cov, target_y_cov, prec=0.1)
```

Comment:
@neerajprad This is a more reasonable test due to two different kernels can have the same output.